src/dags/common/db.py

```diff
@@ -19,6 +19,12 @@ def __init__(self, postgres_conn_id="postgres_default"):
         self.db = self.connection.schema
 
 
+def get_connection(postgres_conn_id: str = "postgres_default") -> Engine:
```

Comment:
This type annotation does not seem correct. `Engine` is not in the type hierarchy of `PostgresHook`

![20210607_083135_screenshot](https://user-images.githubusercontent.com/4290278/120969839-f4a75580-c76a-11eb-9d5d-12878a32f4b4.png)

Reply:
Oeps, pas ik aan